zio-schema/shared/src/main/scala/zio/schema/Schema.scala

```diff
@@ -828,9 +832,11 @@ object Schema {
     override def toString: String = s"Field($label,$schema)"
   }
 
-  sealed trait Record[R] extends Schema[R] {
+  sealed trait Record[R] extends Schema[R] { self =>
     def structure: Chunk[Field[_]]
     def annotations: Chunk[Any] = Chunk.empty
+    // TODO: make this much more robust
+    def defaultValue: Either[String, R] = self.rawConstruct(structure.map(_.schema.defaultValue.getOrElse(())))
```

Comment:
I am not sure what the best method is for lifting a `Chunk[Either[String, A]]` to an `Either[String, Chunk[A]]`.

In `fp-ts`, I would generally use `Traverse` to take care of this, but wasn't sure what the most semantic/idiomatic way to do this in ZIO was. 

I also [asked the question in Discord](https://discord.com/channels/629491597070827530/629491597070827534/854746145191231538) and was informed about `zio-prelude`'s analogous methods, but I wanted to avoid adding additional dependencies to `zio-schema`.

Reply:
I'd suggest avoiding to add ZIO Prelude just yet since it doesn't yet have a 1.0 release. This implementation looks good.